src/MySql/readme.md

```diff
@@ -59,6 +59,9 @@ title: MySQL
 subject-prefix: 'MySQL'
 
 directive:
+  - from: swagger-document
```

Comment:
You are using multiple swagger files. Please specify the swagger file you want to apply the replacement.

Reply:
Will do